edi_835_parser/segments/header_number.py

```diff
@@ -0,0 +1,27 @@
+from edi_835_parser.elements.identifier import Identifier
+from edi_835_parser.elements.integer import Integer
+from edi_835_parser.segments.utilities import split_segment
+
+
+class HeaderNumber:
+	identification = 'LX'
+
+	identifier = Identifier()
+	number = Integer()
+
+	def __init__(self, segment: str):
+		self.index = segment.split(':', 1)[0]
+		segment = segment.split(':', 1)[1]
+
+		self.segment = segment
+		segment = split_segment(segment)
+
+		self.identifier = segment[0]
+		self.number = segment[1]
```

Comment:
Still feel the need for a base class here.

Reply:
fo sho